development/kops/create_configuration.sh

```diff
@@ -81,26 +82,26 @@ data:
     clusterID: $KOPS_CLUSTER_NAME
 EOF
 
-echo "Creating values.yaml"
-cat << EOF > values.yaml
+echo "Creating ./${KOPS_CLUSTER_NAME}/values.yaml"
+cat << EOF > ./${KOPS_CLUSTER_NAME}/values.yaml
 kubernetesVersion: $KUBERNETES_VERSION
 clusterName: $KOPS_CLUSTER_NAME
-configBase: $KOPS_STATE_STORE
+configBase: $KOPS_STATE_STORE/$KOPS_CLUSTER_NAME
 awsRegion: $AWS_DEFAULT_REGION
 EOF
 
 echo "Creating ${KOPS_CLUSTER_NAME}.yaml"
-kops toolbox template --template eks-d.tpl --values values.yaml > "${KOPS_CLUSTER_NAME}.yaml"
+kops toolbox template --template eks-d.tpl --values ./${KOPS_CLUSTER_NAME}/values.yaml > "./${KOPS_CLUSTER_NAME}/${KOPS_CLUSTER_NAME}.yaml"
 
 echo "Creating cluster configuration"
-kops create -f ./"${KOPS_CLUSTER_NAME}.yaml"
+kops create -f "./${KOPS_CLUSTER_NAME}/${KOPS_CLUSTER_NAME}.yaml"
 
 echo "Creating cluster ssh key"
 export SSH_KEY_PATH=${SSH_KEY_PATH:-$HOME/.ssh/id_rsa.pub}
 kops create secret --name $KOPS_CLUSTER_NAME sshpublickey admin -i ${SSH_KEY_PATH}
 
 echo
-echo "# Set these values"
-echo "export AWS_DEFAULT_REGION=$AWS_DEFAULT_REGION"
-echo "export KOPS_CLUSTER_NAME=$KOPS_CLUSTER_NAME"
-echo "export KOPS_STATE_STORE=$KOPS_STATE_STORE"
+echo "# Creating ./${KOPS_CLUSTER_NAME}/env.sh"
+echo "export AWS_DEFAULT_REGION=$AWS_DEFAULT_REGION" | tee ./${KOPS_CLUSTER_NAME}/env.sh
```

Comment:
Should we be checking for AWS_REGION here before setting AWS_DEFAULT_REGION, or keeping it inline with the prow.sh in the other PR?

Reply:
Okay, I updated to it now supports old and new region env vars. Thanks!